source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-03-30 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+[Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) has been a great tool so far in providing a prepackaged development environment filled with production-grade, pre-configured tools, that makes the container developer's lives easier.
+
+However, many of our users experienced that the downloads of the vagrant box `projectatomic/adb` of around 500 MB sometimes take time and even fails to download in a slow networks. This and few other potential issues which I would like to address in my another blog post impelled us to think about the better user experience.
```

Comment:
@budhrg The download size was not the biggest issue with ADB, but the issues which are Vagrant related. I do not see any value this text adds to the overall blog post. We should just tell Minishift is a better tool than ADB  because of overall user experience. I will not suggest you to elaborate on this in this blog